vagga.yaml

```diff
@@ -6,43 +6,30 @@ containers:
     - !Install [alpine-base, py-sphinx, make]
     - !Py2Requirements docs/requirements.txt
 
+  ubuntu-base:
+    setup:
+    - !Ubuntu xenial
+    - !UbuntuUniverse
+    - !Install [build-essential, ca-certificates]
+    - !Install [file]  # dependency of checkinstall (bug #46)
+    - !Install [openssl] # necessary as downloads from static.rust-lang.org need SSL now
```

Comment:
The Tar and TarInstall commands don't use the target system for downloading. It's done by setting intermediary alpine system which has wget and ca-certificates. So you don't need this I think.